bittensor/metagraph.py

```diff
@@ -214,14 +224,79 @@ def uids(self) -> torch.LongTensor:
         return self.state.uids
 
     @property
-    def stake(self) -> torch.LongTensor:
+    def stake(self) -> torch.FloatTensor:
         r""" Returns the stake held by each known neuron.
         Returns
-            stake: (:obj:`torch.LongTensor` of shape :obj:`(n)`):
+            stake: (:obj:`torch.FloatTensor` of shape :obj:`(n)`):
+                stake of each known neuron.
+        """
+        return self.state.stake
+
+    @property
+    def S(self) -> torch.FloatTensor:
```

Comment:
nit: Do we really want to have singular letter function names? It's understood that that is the stake, but we need better naming.

Reply:
reason for this is the relationship between this and the paper